pinc/User.inc

```diff
@@ -555,6 +557,12 @@ function check_username($username, $registering = false)
         return $error;
     }
 
+    // For new registrations, prevent too-short usernames
+    if ($registering && strlen($username) < $username_min_len) {
+        $error = sprintf(_("Your User Name is too short.<br>(The minimum is %s characters.)"), $username_min_len);
```

Comment:
Not that it makes a difference to non-strict PHP, but shouldn't the `%s` here (and above) be `%d`?